crates/uv-cli/src/lib.rs

```diff
@@ -2761,4 +2761,8 @@ pub struct DisplayTreeArgs {
     /// Show the reverse dependencies for the given package. This flag will invert the tree and display the packages that depend on the given package.
     #[arg(long, alias = "reverse")]
     pub invert: bool,
+
+    /// Show the version constraint(s) imposed on each package.
+    #[arg(long)]
```

Comment:
Can we add a short flag for this too?

We should probably use `--show-` not `--emit-` to match the rest of our CLI. `--emit-` is specific to the `pip compile` output.

Reply:
I think a short flag is hard here (`-e` and `-s` are both useful for global things (e.g., we use both in Ruff), it would be a shame to make them blocked on this flag in the future).

Reply:
Yeah maybe "-V" or similar would make sense? I don't think a short-flag for the "emit" or "show" portion makes sense.

Reply:
Apparently `-V` is in use by `--version` already (fails at runtime), interesting.